packaging/systemd/poet2

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+SAWTOOTH_POET2_ARGS=-v -C tcp://localhost:5050
```

Comment:
Would this not necessitate the validator to run on the same machine & port?

Reply:
I can comment this out if the poet2 application is connecting to  some default address and port

Reply:
The [check](https://github.com/askmish/sawtooth-poet2/blob/master/src/core/src/main.rs#L69) is in place but need to be fixed.